app/helpers/tags_helper.rb

```diff
@@ -1,13 +1,14 @@
 module TagsHelper
 
-  def tag_type_label(tag_type)
+  def tag_type_class(tag_type)
```

Comment:
Would it make sense to move this `tag_type_class(tag_type)` method to the new tag_type_helper.rb?

Reply:
Yes defo.